README.md

```diff
@@ -624,130 +650,155 @@ Note that it is also possible to generate a synthetic dataset based on the entir
+| `simple`                           | MacOS                | Model saved in `<model_name>-mlx-q` directory.                                                                           |
 | `full`                             | Linux & MacOS        | `.bin` and `.gguf` models saved in `~/.local/share/instructlab/checkpoints/hf_format` directory. Two models in each `sample_*` directory: one quantized (`Q4-M-K` suffix) and one full precision. |
 | `accelerated`                      | Linux                | Models saved in `~/.local/share/instructlab/checkpoints`. Can be evaluated with `ilab model evaluate` to choose the best one. |
 | `lab-multiphase`                   | Linux                | Phase 1 models saved in `~/.local/share/instructlab/phased/phase1/checkpoints` (Knowledge training). Phase 2 models saved in `~/.local/share/instructlab/phased/phase2/checkpoints` (Skills training). Evaluation is run for both phases to identify the best checkpoint. |
 
+#### Train the model locally
+
+- Train the model with your synthetic data with the `ilab model train` command:
+
+   ```shell
+   ilab model train
+   ```
+
+   ⏳ This step can potentially take **several hours** to complete depending on your computing resources. Please stop `ilab model chat` and `ilab model serve` first to free resources.
+
 When running multi phase training evaluation is run on each phase, we will tell you which checkpoint in this folder performs the best.
 
 #### Train the model locally on an M-series Mac or on Linux using the full pipeline
 
-To train the model locally on your M-Series Mac using our full pipeline and MPS or on your Linux laptop/deskop using CPU:
+- To train the model locally on your M-Series Mac using our full pipeline and MPS or on your Linux laptop/desktop using CPU:
 
-```shell
-ilab model train --pipeline full --device mps
-```
+   ```shell
+   ilab model train --pipeline full --device mps --data-path <path-to-sdg-dataset>
+   ```
 
-```shell
-ilab model train --pipeline full --device cpu
-```
+   ```shell
+   ilab model train --pipeline full --device cpu --data-path <path-to-sdg-dataset>
+   ```
 
-> **Note:** ⏳ This process will take a while to complete. If you run for ~8 epochs it will take several hours.
+   *Example command*
 
-`ilab model train` outputs a directory for each epoch that resembles the following structure:
+   ```shell
+   ilab model train --pipeline full --device cpu --data-path ~/.local/share/instructlab/datasets/knowledge_train_msgs_2024-10-23T09_14_44.jsonl
+   ```
 
-```shell
-$ ls ~/.local/share/instructlab/checkpoints/hf_format/samples_0/
-├── added_tokens.json
-├── config.json
-├── pytorch_model.bin
-├── pytorch_model.gguf
-├── pytorch_model-Q4_K_M.gguf
-├── special_tokens_map.json
-├── tokenizer.json
-├── tokenizer_config.json
-└── tokenizer.model
-```
+   ⏳ This process will take a while to complete. If you run for ~8 epochs it will take several hours.
```

Comment:
How do I tune it down to say 1 epoch for fastness?

Reply:
I'll see if there is a flag but users can also adjust it in the config